src/Python/tests/tests.py

```diff
@@ -381,15 +463,14 @@ def test_metadata(self):
                               "name":"Behavioural Genomics" },
                        "who":"Firstname Lastname",
                        "timestamp":"2012-04-23T18:25:43.511Z",
-                       "temperature":{ "experiment":22,
-                                       "cultivation":20, "units":"C" },
-                       "humidity":{ "value":40, "units":"%" },
+                       "temperature":23.8,
+                       "humidity":40.3,
                        "dish":{ "type":"petri", "size":35, "units":"mm" },
                        "food":"none",
                        "media":"agarose",
                        "sex":"hermaphrodite",
                        "stage":"adult",
-                       "age":"18:25:43.511",
+                       "age":33.4,
```

Comment:
Better, but this is still super-young for an adult.  How about we double it?  66.8 or something should be comfortably into adulthood at 23.8C. Sorry to be so pedantic, but I think it makes a better impression on worm biologists if the examples make biological sense.